cluster/saltbase/salt/kubelet/default

```diff
@@ -50,8 +50,8 @@
 {% set config = "--config=/etc/kubernetes/manifests" -%}
 
 {% set manifest_url = "" -%}
-{% if grains['roles'][0] == 'kubernetes-master' and grains.cloud in ['gce'] -%}
-  {% set manifest_url = "--manifest-url=http://metadata.google.internal/computeMetadata/v1/instance/attributes/google-container-manifest --manifest-url-header=Metadata-Flavor:Google" -%}
```

Comment:
Consistency is more important to me. I followed the same pattern we have for other parameters, for example, enable_cluster_dns.